.golangci.yaml

```diff
@@ -51,6 +51,8 @@ linters-settings:
   govet:
     enable:
       - nilness
+  goimports:
+    local-prefixes: github.com/cilium/cilium
```

Comment:
Why not "github.com/cilium"?

Reply:
Two reasons:
* "Third party" packages are traditionally packages outside the current repo.
* `github.com/cilium` is not a real Go module, whereas `github.com/cilium/cilium` is.